spec.html

```diff
@@ -16297,7 +16297,7 @@ <h1>Expression Statement</h1>
     <h2>Syntax</h2>
     <emu-grammar type="definition">
       ExpressionStatement[Yield, Await] :
-        [lookahead &lt;! {`{`, `function`, `async` [no |LineTerminator| here] `function`, `class`, `let [`}] Expression[+In, ?Yield, ?Await] `;`
+        [lookahead &lt;! {`{`, `function`, `async` [no |LineTerminator| here] `function`, `class`, `let` `[`}] Expression[+In, ?Yield, ?Await] `;`
```

Comment:
note the NLT requirement between `async` and `function` - might this change allow:
```js
let
[
```
where currently it must be `let [`? (grammar isn't my strong suit, so please correct me if i'm reading this wrong)

Reply:
It currently matches
```js
let
[
```
, and should continue to. (This is so that 
```js
let
[x] = [x]
```
is a variable declaration.)

Reply:
@ljharb This doesn't affect NLT requirement, it would be stated explicitly either way.

Reply:
But this kind of potential questions is exactly the reason I think it's worth fixing this construct to align with others :)

Reply:
Thanks for clarifying; this LGTM but I'm going to defer to other editors.